pkg/services/ngalert/state/historian/annotation.go

```diff
@@ -112,10 +113,27 @@ func buildAnnotationTextAndData(rule *ngmodels.AlertRule, currentState *state.St
 		for _, k := range keys {
 			values = append(values, fmt.Sprintf("%s=%f", k, currentState.Values[k]))
 		}
-		jsonData.Set("values", simplejson.NewFromAny(currentState.Values))
+		jsonData.Set("values", jsonifyValues(currentState.Values))
```

Comment:
I wonder if we can embed EvalString instead? We can parse it on read when it is needed.. Just an idea

Reply:
I looked into parse-on-read, unfortunately it's a very large amount of code churn. The annotations service has simplejson deeply baked into its API, and it relies on serialization to happen at the ORM level. It would be a pretty deep change - changing the annotation service's models in a fundamental way.